src/operator.py

```diff
@@ -241,6 +248,14 @@ def register_drone(request, response):
     print("Calling register_drone service...")
     drone_name = request["drone_name"]
     drone_type = request["drone_type"]
+
+    if drone_name in drone_names:
+        response["success"] = False
+        response["message"] = "A drone with this name already exists."
+        response["id"] = 0
+        print("A drone with this name already exists")
+        return True
```

Comment:
Does return False not work? I don't remember if its True only on service success or just always.

Reply:
I tried returning False and the service just hung. So I think it has to return True.